src/js/view/controls/components/timeslider.js

```diff
@@ -106,6 +106,12 @@ class TimeSlider extends Slider {
             .change('buffer', this.onBuffer, this)
             .change('streamType', this.onStreamType, this);
 
+        // Clear cues on player model's playlistItem change event
+        const { _model } = this._model;
+        _model.change('playlistItem', (model) => {
```

Comment:
`this._model.player` get's the player view model. You should not access the internal model directly.

Reply:
`this._model.player.on('change:playlistItem', this.onPlaylistItem, this);`

You'll need to update the 'Control Bar' tests as well since they use a mock SimpleModel that will need a player property added.